docs/manuals/source/include/config/DirSelectionPatterns-sql1.conf

```diff
@@ -0,0 +1,19 @@
+
+# SQL selecting all jobid in pool "Full"
+# which is a terminated backup (T,W)
+# and was never copied or migrated (PriorJobid 0).
+
+Job {
+    Name = "<name>"
+    JobDefs = "DefaultJob"
+    Type = Migrate
+    Selection Type = SQL Query
+    # Multiple lines instructions is available since version 21.0.0
```

Comment:
```suggestion
```
As this documentation is only for recent releases, we don't need this information.

Reply:
I would prefer to keep it, this one comment line will avoid support report about why is this failing when copy and pasted.